app/assets/javascripts/discourse/app/components/sidebar/user/custom-sections.js

```diff
@@ -2,30 +2,53 @@ import Component from "@glimmer/component";
 import { inject as service } from "@ember/service";
 import { ajax } from "discourse/lib/ajax";
 import { bind } from "discourse-common/utils/decorators";
-import Section from "discourse/components/sidebar/user/section";
+import Section from "discourse/lib/sidebar/section";
+import CommunitySection from "discourse/lib/sidebar/community-section";
 
 export default class SidebarUserCustomSections extends Component {
   @service currentUser;
   @service router;
   @service messageBus;
+  @service appEvents;
+  @service topicTrackingState;
+  @service siteSettings;
 
   constructor() {
     super(...arguments);
     this.messageBus.subscribe("/refresh-sidebar-sections", this._refresh);
+    this.cacheSections;
```

Comment:
I don't think you need this line here. Fields can be initialised in the class declaration like so:

```
class Something {
  cacheSections: [];
}
```